Core/OxidEE_Events.php

```diff
@@ -160,38 +157,26 @@ private static function _createOrderTable()
     private static function _createOrderTransactionTable()
     {
         $sQuery = "CREATE TABLE IF NOT EXISTS `wdoxidee_ordertransactions`(
-            `wdoxidee_transactionid` int(11) unsigned NOT NULL AUTO_INCREMENT,
-            `wdoxidee_orderid` char(32) character set latin1 collate latin1_general_ci NOT NULL,
-            `wdoxidee_requestid` varchar(256) NOT NULL,
-            `wdoxidee_transactiontype` varchar(128) NOT NULL,
-            `wdoxidee_amount` decimal(9,2) NOT NULL,
-            `wdoxidee_refundedamount` decimal(9,2) NOT NULL,
-            `wdoxidee_currency` varchar(32) NOT NULL,
-            `wdoxidee_transactiondate` datetime NOT NULL,
-            `wdoxidee_transactionstatus` enum('pending','success','error') NOT NULL DEFAULT 'pending',
-            `wdoxidee_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
-            `wdoxidee_comment` mediumtext,
-            PRIMARY KEY (`wdoxidee_transactionid`),
-            KEY `wdoxidee_orderid` (`wdoxidee_orderid`),
-            KEY `wdoxidee_transactiondate` (`wdoxidee_transactiondate`)
+            `OXID` char(32) NOT NULL,
+            `WDOXIDEE_ORDERID` varchar(32) NOT NULL,
+            `WDOXIDEE_TRANSACTIONID` varchar(32) NOT NULL,
+            `WDOXIDEE_PARENTTRANSACTIONID` varchar(32),
+            `WDOXIDEE_REQUESTID` varchar(32) NOT NULL,
+            `WDOXIDEE_ACTION` enum('purchase','authorization') NOT NULL DEFAULT 'purchase',
+            `WDOXIDEE_STATE` enum('success','error','pending'),
+            `WDOXIDEE_PAYMENTMETHOD` varchar(32) NOT NULL,
+            `WDOXIDEE_AMOUNT` double NOT NULL,
+            `WDOXIDEE_CURRENCY` varchar(32) NOT NULL,
+            `WDOXIDEE_DESCRIPTOR` varchar(32) NOT NULL,
```

Comment:
I also don't see the use of this as it is in the response XML, you can read it from there.

Reply:
We created individual fields so that we don't have to parse XML all the time, and again because of the transaction detail screen from OpenCart (WDPD-27). Is it okay if we keep it?

https://jira.parkside.at/browse/WDPD-27

Reply:
In the detail view you will have to pars the date i see no way around it. You see on the screen shot that there are data as account holder, shipping, credit card and so on.

Reply:
Good point. We changed it to work just like in OpenCart. We now only have individual fields for those values that we need to display in the table overview.